src/utils/svgicons.tsx

```diff
@@ -0,0 +1,182 @@
+import React from "react";
```

Comment:
You can use "camelCase" for naming the file.

Reply:
done